web/angular-wallet/src/app/shared/pipes/burst-amount.pipe.ts

```diff
@@ -13,19 +14,22 @@ export class BurstAmountPipe implements PipeTransform {
   constructor(private i18nService: I18nService) {
   }
 
-  private static hasArgument(args: any[], argName: string): boolean {
-    return args.indexOf(argName) !== -1;
-  }
-
   private shouldUpdate(): boolean {
     return this.i18nService.currentLanguage.code !== this.cachedLanguageCode;
   }
 
-  transform(value: string | number, ...args: any[]): string {
+  transform(value: string | number,
+            inputType: 'planck' | 'burst' = 'burst',
+            isShortForm: boolean = false,
```

Comment:
due to IDE support it's ok. Otherwis, I'd prefer `...args` for better readability